packages/vector_graphics/pubspec.yaml

```diff
@@ -4,7 +4,7 @@ repository: https://github.com/flutter/packages/tree/main/packages/vector_graphi
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+vector_graphics%22
 # See https://github.com/flutter/flutter/issues/157626 before publishing a new
 # version.
```

Comment:
This PR ignored the TODO here, so `vector_graphics` is now mis-aligned with the `vector_graphics_*` versions. @jonahwilliams We need a versioning plan here ASAP.

Reply:
💩 – sorry, folks

Reply:
I thought these weren't being auto published yet?

Reply:
I got confirmed so I flipped the bit and assumed the robots would "do the right thing" 🤷

Reply:
> I thought these weren't being auto published yet?

I need to publish an initial version so that the pub.dev metadata (issue link, source link, etc.) would be correct. That literally landed yesterday at the end of my workday; I didn't expect someone to create a PR, ignore the comment on the version saying not to publish a new version yet, get it approved, and land it, all before I started my next day and could do a follow-up PR to mark them as unpublished.